runtime/calamari/src/assets_config.rs

```diff
@@ -194,5 +194,11 @@ impl pallet_manta_sbt::Config for Runtime {
     type MintsPerReserve = ConstU16<5>;
     type ReservePrice = ConstU128<{ 100_000 * KMA }>;
     type SbtMetadataBound = ConstU32<300>;
+    type ChainId = ConstU64<2084>;
```

Comment:
This chainId configuration will not work when user use Metamask to sign message, because Metamask need to make sure the chainId is evm based. 
- Make this as dynamic configuration set by AdminOrigin for different MintType, because different evm chain has different chainId. 
- But seems the chainId hasn't play important role in the verification process, maybe we can set it to Ethereum mainet chainId.